3rdparty/dlmalloc/allocator.c

```diff
@@ -21,15 +18,6 @@
 #pragma GCC diagnostic ignored "-Wparentheses-equality"
 #endif
 
-OE_INLINE
-int sched_yield(void)
```

Comment:
Why was this removed? Is it not needed anymore?

Reply:
It's not needed anymore because `corelibc/sched.h` defines it (previously dlmalloc did not use corelibc headers at all).